services/site/.env.example

```
@@ -142,17 +137,18 @@ R2_SECRET_ACCESS_KEY=MOCKR2SECRETACCESSKEY
 # access key id opts in (starts with `MOCK`).
 CALL_KENT_R2_BUCKET=MOCK_CALL_KENT_R2_BUCKET
 # Feature: /calls/admin (offloaded FFmpeg audio generation from the app)
-# Required (Fly app env):
+# Required (site app env):
 # - Cloudflare Queue identifier for `kcd-call-kent-audio`.
 # - Get it from Cloudflare dashboard -> Queues -> queue details.
 CALL_KENT_AUDIO_CF_QUEUE_ID=replace-with-cloudflare-queue-id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CALL_KENT_AUDIO_CF_QUEUE_ID key should go before the CALL_KENT_R2_BUCKET key

(UnorderedKey)